packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/en.ftl

```diff
@@ -10,3 +10,11 @@ sub-change-submit = Confirm change
 sub-update-current-plan-label = Current plan
 sub-update-new-plan-label = New plan
 sub-update-total-label = New total
+sub-update-one-time-charges = One time charges (due today)
+sub-update-prorated-total = <strong>Upgrade</strong> (Prorated to end of period)
+
+# Subscription plan change interval
```

Comment:
```suggestion
## Frequency of payment for subscription plan change
```

Use a group-level comment (two dashes) if it applies to multiple strings, then reset it after the group with `##`. We should also make it clearer that it related to frequency of payment.